### PR TITLE
[nrf-ble-driver] Fix version errors

### PR DESCRIPTION
**Pull request by General-zjs**
[nrf ble driver] Fix the inconsistency between the source code version and the output version

### CODE REVIEW

**Comment by Adela0814** [comment]
@h1562075881  Please upgrade the `port-version` via `vcpkg x-add-version nrf-ble-driver`.

File: ports/nrf-ble-driver/portfile.cmake [hunk 1]
```diff
@@ -6,10 +6,11 @@ These can be installed on Ubuntu systems via sudo apt install libudev-dev"
     )
 endif()
 
+set(NRF_BLE_VERSION 4.1.4)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO NordicSemiconductor/pc-ble-driver
-    REF v4.1.4
+    REF "v${NRF_BLE_VERSION}"
```
Comment (autoantwort) [inline ports/nrf-ble-driver/portfile.cmake:13]: You can use `${VERSION}` instead

**Comment by Adela0814** [comment]
Convert this PR to draft since there is no progress. Please ping us if this PR is ready for review again.

**Comment by Adela0814** [comment]
Closing this PR since it seems that no progress is being made. Please ping us to reopen if work is still being done.

**Pull request closed without merging**